benchmark/fluid/kube_gen_job.py

```diff
@@ -0,0 +1,332 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+import copy
+import argparse
+import random
+import os
+
+pserver = {
+    "apiVersion": "extensions/v1beta1",
```

Comment:
And, In my view, the yaml format is more concise than json format? which one do you prefer?

Reply:
Comments all done, I kept using json so that we can directly import it as in-memory data.